### PR TITLE
ran a formatter on the codebase

### PR DESCRIPTION
**Pull request by PolarianDev**
Intellij must really not like your formatting :P

### CODE REVIEW

File: src/main/java/vip/floatationdevice/guilded4j/G4JClient.java [hunk 1]
```diff
@@ -12,172 +12,170 @@
...
 
 /**
  * Guilded4J's main component, can send HTTP requests to the API, receive WebSocket events and post them as {@link vip.floatationdevice.guilded4j.event.GuildedEvent}s.
+ *
  * @see G4JWebSocketClient
  * @see vip.floatationdevice.guilded4j.event.GuildedEvent
  */
-public class G4JClient
-{
+public class G4JClient {
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:28]: don't

File: src/main/java/vip/floatationdevice/guilded4j/G4JClient.java [hunk 2]
```diff
@@ -12,172 +12,170 @@
...
      */
     public G4JWebSocketClient ws;
+    String authToken;
+    int httpTimeout = 20000;
 
-    public G4JClient(String authToken)
-    {
-        this.authToken=authToken;
-        ws=new G4JWebSocketClient(authToken);
+    public G4JClient(String authToken) {
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:55]: don't

File: src/main/java/vip/floatationdevice/guilded4j/G4JClient.java [hunk 3]
```diff
@@ -12,172 +12,170 @@
...
+        this.authToken = authToken;
+        ws = new G4JWebSocketClient(authToken);
     }
 
-    public G4JClient(String authToken, String lastMessageId)
-    {
-        this.authToken=authToken;
-        ws=new G4JWebSocketClient(authToken, lastMessageId);
+    public G4JClient(String authToken, String lastMessageId) {
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:60]: don't

File: src/main/java/vip/floatationdevice/guilded4j/G4JClient.java [hunk 4]
```diff
@@ -12,172 +12,170 @@
...
 //============================== API FUNCTIONS START ==============================
 ////////////////////////////// Chat & messaging //////////////////////////////
 
     /**
      * Create a channel message.<br>
      * <a href="https://www.guilded.gg/docs/api/chat/ChannelMessageCreate" target=_blank>https://www.guilded.gg/docs/api/chat/ChannelMessageCreate</a>
-     * @param channelId The ID of the channel.
-     * @param content The message content to create.
+     *
+     * @param channelId       The ID of the channel.
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:71]: too many spaces

File: src/main/java/vip/floatationdevice/guilded4j/G4JClient.java [hunk 5]
```diff
@@ -12,172 +12,170 @@
...
+     * @param content         The message content to create.
      * @param replyMessageIds The ID of the message(s) to reply to.
-     * @param isPrivate If set, this message will only be seen by those mentioned or replied to.
+     * @param isPrivate       If set, this message will only be seen by those mentioned or replied to.
      * @return The newly created message's ChatMessage object.
-     * @throws GuildedException if Guilded API returned an error JSON string.
+     * @throws GuildedException                     if Guilded API returned an error JSON string.
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:76]: tf

File: src/main/java/vip/floatationdevice/guilded4j/G4JClient.java [hunk 6]
```diff
@@ -12,172 +12,170 @@
...
     }
 
     /**
      * Delete a channel message.<br>
      * <a href="https://www.guilded.gg/docs/api/chat/ChannelMessageDelete" target=_blank>https://www.guilded.gg/docs/api/chat/ChannelMessageDelete</a>
+     *
      * @param channelId The ID of the channel.
      * @param messageId The ID of the message.
-     * @throws GuildedException if Guilded API returned an error JSON string.
+     * @throws GuildedException                     if Guilded API returned an error JSON string.
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:100]: ah sh!t here we go again

Comment (PolarianDev) [reply src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:100]: the formatter must really hate you

File: src/main/java/vip/floatationdevice/guilded4j/G4JClient.java [hunk 7]
```diff
@@ -186,108 +184,98 @@ public ChatMessage getMessage(String channelId, String messageId)
...
             }
-        }
-        else
-        {
-            result=new JSONObject(HttpRequest.put(NICKNAME_URL_OLD.replace("{userId}",userId)).
-                    header("Authorization","Bearer "+authToken).
-                    header("Accept","application/json").
-                    header("Content-type","application/json").
-                    body(new JSONObject().set("nickname",nickname).toString()).
+        } else {
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JClient.java:271]: image0-1

File: src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java [hunk 8]
```diff
@@ -23,84 +23,93 @@
...
-     * See {@link G4JDebugger} for example uses.
-     */
-    public EventBus eventBus=new EventBus();
+    private static URI initURI() {
+        try {
+            return new URI("wss://api.guilded.gg/v1/websocket");
+        } catch (URISyntaxException e) {/*this is impossible*/
+            return null;
+        }
+    }
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java:70]:
```java
try
{
  ;
}
catch()
{
  ;
}
```

File: src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java [hunk 9]
```diff
@@ -23,84 +23,93 @@
...
      * Toggle printing the received JSON string from WebSocket to stdout.
      * Disabled (dump=false) by default.
+     *
      * @return Dumping status after changed.
      */
-    public boolean toggleDump(){dump=!dump;return dump;}
+    public boolean toggleDump() {
+        dump = !dump;
+        return dump;
+    }
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java:81]:
```java
function name()
{
  ;
}
```

Comment (PolarianDev) [reply src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java:81]: this isn't C#

File: src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java [hunk 10]
```diff
@@ -23,84 +23,93 @@
...
 
     /**
      * No current use.
      */
     @Override
-    public void onOpen(ServerHandshake h) {}
+    public void onOpen(ServerHandshake h) {
+    }
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java:88]: E

File: src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java [hunk 11]
```diff
@@ -23,84 +23,93 @@
...
+        JSONObject json = new JSONObject(rawMessage);
+        if (dump) System.out.println("\n" + json.toStringPretty());
+        int op = json.getInt("op");//operation code: 0, 1, 2, 8, 9
+        String eventID = json.getStr("s");//event id (aka. lastMessageId)
+        String eventType = json.getStr("t");//hope they wont change this key name in the future
+        String serverID = (String) json.getByPath("d.serverId");// null if op != 0
 
-        switch(op)
-        {
+        switch (op) {
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/G4JWebSocketClient.java:106]:
```java
switch()
{
  case:
  {
    ;
  }
}
```

File: src/main/java/vip/floatationdevice/guilded4j/enums/ExceptionType.java [hunk 12]
```diff
@@ -11,13 +11,30 @@
...
+    /**
+     * 409
+     */
+    CONFLICTED,
+    /**
+     * 500-504
+     */
+    INTERNAL_SERVER_ERROR,
     OTHER
 }
```
Comment (MCUmbrella) [inline src/main/java/vip/floatationdevice/guilded4j/enums/ExceptionType.java:40]: barrier
# nothing approved in this file

Comment (PolarianDev) [reply src/main/java/vip/floatationdevice/guilded4j/enums/ExceptionType.java:40]: its neater though

**Comment by MCUmbrella** [comment]
in a nutshell
```java
class
{
  function name()
  {
    ;
    ;
  }
  function name(){;}
  for()
  {
    ...
  }
  if()
  {
    ...
  }
  else if()
  {
    ...
  }
  else
  {
    ...
  }
  switch()
  {
    case:
    {
      ...
    }
    default:
    {
      ...
    }
  }
  try
  {
    ...
  }
  catch()
  {
    ...
  }
}
```
one more thing
lambda-free

**Pull request closed without merging**